README.md

```diff
@@ -292,7 +313,36 @@ _extra_paths:
   - ~/Projects/templates
 ```
 
-### Include other yaml files
+##### Arguments defaults
+
+The keys `_exclude`, `_skip_if_exists`, `_tasks`, and `_extra_paths` in the `copier.yml`
+file, will be treated as the default values for the `exclude`, `tasks`, and ,
+`extra_paths` arguments to `copier.copy()`.
```

Comment:
I think this is better in a list:

- argument (copier.yml key): usage
- ...

Because we should expect it to grow.

Reply:
You mean describe all keys and their corresponding CLI arguments in a list/table? All the keys are already explained in the example block, maybe we could simply add their corresponding CLI arguments there.

Reply:
OK, just try to keep that maintainable.

Reply:
Alright, added the corresponding CLI/API options to the examples and removed that paragraph (that I did not write but just moved by the way!)